.github/scripts/rollback/delete-github-release-and-tag.js

```diff
@@ -0,0 +1,64 @@
+const { Octokit } = require('@octokit/rest');
+
+async function main() {
+  const {
+    GITHUB_TOKEN,
+    REPO_OWNER,
+    REPO_NAME,
+    ROLLBACK_VERSION,
+  } = process.env;
+
+  if (!GITHUB_TOKEN || !REPO_OWNER || !REPO_NAME || !ROLLBACK_VERSION) {
+    throw new Error(
+      'GITHUB_TOKEN, REPO_OWNER, REPO_NAME, and ROLLBACK_VERSION env vars are required'
+    );
+  }
+
+  const octokit = new Octokit({ auth: GITHUB_TOKEN });
+
+  const tagVariants = [ROLLBACK_VERSION];
+
+  // Delete matching releases
+  const { data: releases } = await octokit.repos.listReleases({
+    owner: REPO_OWNER,
+    repo: REPO_NAME,
+    page: 1, // only last release is needed, so fetching only 1st page
+  });
+
+  const rel = releases.find(r =>
```

Comment:
Is there a possibility that we could get an array of results for the find method ?
For example if the first page has result tags ` v1.0.0` and `v1.0.0-rc` and input is `ROLLBACK_VERSION=v1.0.0`